todo-app/src/Components/ToDo.tsx

```diff
@@ -0,0 +1,15 @@
+import IToDo from "../interfaces/Todo";
+
+const ToDo = ({ handleCheck, deleteToDo, item, index }: { handleCheck: (index: number) => void, deleteToDo: (index: number) => void, item: IToDo, index: number }) => {
```

Comment:
Type we. can define separately for e.g

interface IProps {
    handleCheck: (index: number) => void,
    .....
}
const ToDo = (props: IProps) => {
const { handleCheck } = props;
}